docs/installing.rst

```diff
@@ -32,21 +34,21 @@ Then run the following:
 
 .. warning::
 
-   Be cautious if you're using a Python install that's managed by your operating
-   system or another package manager. get-pip.py does not coordinate with
+   Be cautious if you are using a Python install that is managed by your operating
+   system or another package manager. ``get-pip.py`` does not coordinate with
    those tools, and may leave your system in an inconsistent state.
 
-get-pip.py will also install :ref:`pypug:setuptools` [3]_ and :ref:`pypug:wheel`,
-if they're not already. :ref:`pypug:setuptools` is required to install
+``get-pip.py`` also installs :ref:`pypug:setuptools` [3]_ and :ref:`pypug:wheel`. 
+If they are not already, :ref:`pypug:setuptools` is required to install
```

Comment:
Your re-positioning of the comma and period malforms the meaning of both sentences, this needs to be switched back.